.env.example

```diff
@@ -1,5 +1,5 @@
 # Supabase
-NEXT_PUBLIC_SUPABASE_URL=http://localhost:54321
+NEXT_PUBLIC_SUPABASE_URL=http://kong:8000
```

Comment:
I haven’t actually let it run yet, I get heartbroken by failed deployments 🤣

I only changed to that since Kong will be the supabase url, which one works for you?

Reply:
By default if you use the Supabase CLI, it runs the various services in docker containers and exposes `http://localhost:54321` as the URL to be used here - this is also the case in the way it's run in `docker-compose` in this PR (applying suggestion below to resolve!)